pkg/statsd/handler_http_forwarder_v2.go

```diff
@@ -308,7 +308,7 @@ func (hfh *HttpForwarderHandlerV2) sendNop(ctx context.Context) {
 
 func (hfh *HttpForwarderHandlerV2) Run(ctx context.Context) {
 	var wg wait.Group
-	wg.StartWithContext(ctx, hfh.sendNop)
+	hfh.sendNop(ctx)
```

Comment:
Just for my context, why do we need to send a nop before we do actual flushes?

Reply:
The deepcheck checks if we've ever sent metrics - if we can't send metrics, we don't want to start.

We don't check how longs it's been though, since this is about misconfiguration, not backend going down.